docs/default_csi_plugin.md

```diff
@@ -0,0 +1,80 @@
+# MicroShift Default CSI Plugin
+
+MicroShift enables dynamic storage provisioning out of the box with the ODF-LVM CSI plugin. This plugin is downstream
+Red Hat distribution of TopoLVM. This feature is not currently configurable but will be in the near future. For more
+information on ODF-LVM, visit the repo's [README](https://github.com/red-hat-storage/topolvm).
+
+## Design
+
+ODF-LVM manifests can be found in [microshift/assets/components/odf-lvm](../assets/components/odf-lvm). For an overview
+of ODF-LVM architecture, see the [design doc](https://github.com/red-hat-storage/topolvm/blob/main/docs/design.md).
+
+## Deployment
+
+ODF-LVM is deployed onto the cluster in the `openshift-storage` namespace, after MicroShift boots. In order to reduce
+MicroShift's storage footprint and cluster overhead,
```

Comment:
This statement about capacity tracking would fit in well in the enhancement document to explain the choice. Here I think it's simplest to focus on the outcome from the user's perspective: "ODF-LVM uses StorageCapacity tracking to ensure that pods are not schedule if the storage they request is not available" (I think that's what it does?)